lib/dashboard.dart

```diff
@@ -40,6 +43,20 @@ class _DashboardState extends State<Dashboard> {
       widgets.add(const BackupSeedCard());
     }
 
+    final fakeStuff = Container(
```

Comment:
🔧 I wouldn't call the widget fakeStuff 🙈 Ideally you can separate the data from the presentation.. fake payments / data etc. is fine. But the widget should work the same with mock data as well as real data alike.. 😉 

```suggestion
    final paymentHistory = Container(
```